examples/gpt/README.md

```diff
@@ -0,0 +1,62 @@
+# GPT Agent
+
+This example uses the [OpenAI](https://openai.com) API for GPT-based responses and the [XMTP](https://xmtp.org) protocol for secure messaging. You can test your agent on [xmtp.chat](https://xmtp.chat) or any other XMTP-compatible client.
```

Comment:
we can just say XMTP here. XMTP protocol is redundant since the P stands for protocol.